third_party/terraform/website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -40,11 +46,21 @@ resource "google_cloudfunctions_function" "function" {
   labels = {
     my-label = "my-label-value"
   }
-  
+
   environment_variables = {
     MY_ENV_VAR = "my-env-var-value"
   }
 }
+
+# Add IAM member for a user who can invoke the function (no admin actions)
+resource "google_cloudfunctions_function_iam_member" "invoker" {
```

Comment:
I'd like secure cloud functions to be our default example, but what do you think of adding a second example that shows how to add a publicly invokable fn? That example will simulate the current behaviour for existing users, and I've seen users struggle setting `allUsers` permissions in the past.

Reply:
done!